### ISSUE TITLE
A11y 1.1 - Grammar Improvements 2

### ISSUE THREAD

**Issue by mbeganyi** [mattgarrish/epub-revision#1838]
Hi all,

Just a few additional notes regarding grammar in the spec:

> The specification also establishes how to identify content that EPUB Creators have optimized for specific user needs so cannot meet broad accessibility requirements.

Not sure exactly what the recommendation would be here, but doesn't come across clearly. Perhaps something about content meeting broad accessibility requirements?

> ...it is often not enough on its own to ensure that the text is fully accessible in any given language (e.g., additional information about directionality, emphasis, pronunciation, etc. may also be neeed).

"Neeed" at end of sentence

> Inserting page breaks markers into an EPUB Publication provides users with context about where they are in the text. 

"page breaks markers"

**Comment by mattgarrish** [comment]
> > The specification also establishes how to identify content that EPUB Creators have optimized for specific user needs so cannot meet broad accessibility requirements.
> 
> Not sure exactly what the recommendation would be here, but doesn't come across clearly.

Yes, there are two separate thoughts merged together here. What it's generally getting at is that a publication that is optimized for one way of reading cannot, by definition, be broadly accessible to all users. The whole paragraph needs some tweaking, though, as we've changed the reference section.

Thanks for the reports!
